add-on/panels/js/panel.jsx

```diff
@@ -713,10 +708,12 @@ loop.panel = (function(_, mozL10n) {
      */
     _renderLoadingRoomsView: function() {
       return (
-        <div className="room-list">
+        <div className="rooms">
```

Comment:
Please add an XXX comment that we should factor this apart so that we're duplicating the rooms element in a bunch of different places.